content/posts/linux-scheduler-journey.md

```diff
@@ -0,0 +1,600 @@
+---
+title: "A journey into the Linux scheduler"
+date: 2022-06-01T09:00:00+02:00
+tags: [linux, scheduler]
+categories: [linux]
+draft: false
+---
+
+Two years ago more or less I started my journey in Linux. I was scared at first and I didn't know where to start from.
+But then I decided to buy a [book](https://www.amazon.com/Linux-Kernel-Development-Robert-Love/dp/0672329468) - and what a book! - in order to follow a path.
+
+Along the way I integrated the material with up-to-date documentation from [kernel.org](https://docs.kernel.org) and [source code](https://elixir.bootlin.com/linux/v5.17.9/source). In the meantime I started to learn C a bit so that I also could have [played](https://github.com/maxgio92/linux/tree/syscall/maxgio) with what I was learning, step by step.
```

Comment:
```suggestion
Along the way, I integrated the material with up-to-date documentation from [kernel.org](https://docs.kernel.org) and [source code](https://elixir.bootlin.com/linux/v5.17.9/source). In the meantime, I started to learn C a bit so that I also could have [played](https://github.com/maxgio92/linux/tree/syscall/maxgio) with what I was learning, step by step.
```